File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1066.c
#include <stdio.h>

int main(){
    int a, pos=0, neg=0, par=0, imp=0, i;
    for(i=0;i<5;i++){
        scanf("%d", &a);
        if(a>0){
            pos++;
        } else if(a<0){
            neg++;
        }
        if(a%2==0){
            par++;
        } else{
            imp++;
        }
    }
    printf("%d valor(es) par(es)\n", par);
    printf("%d valor(es) impar(es)\n", imp);
    printf("%d valor(es) positivo(s)\n", pos);
    printf("%d valor(es) negativo(s)\n", neg);

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1066.rb
pos=0
neg=0
par=0
imp=0

for i in 1..5 do
    a = gets.to_i
    if(a>0) then
        pos=pos+1
    elsif(a<0) then
        neg=neg+1
    end
    if(a%2==0) then
        par=par+1
    else
        imp=imp+1
    end
end

puts "#{par} valor(es) par(es)"
puts "#{imp} valor(es) impar(es)"
puts "#{pos} valor(es) positivo(s)"
puts "#{neg} valor(es) negativo(s)"
